main.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
```

Comment:
Need to have dependencies saved in requirements.txt . I had to install them myself to execute your code.

To isolate your dependencies
pip freeze > requirements.txt